tests/integrations/raftstore/test_replica_read.rs

```diff
@@ -208,33 +240,40 @@ fn test_read_hibernated_region() {
     assert!(!resp2.get_header().has_error(), "{:?}", resp2);
 }
 
-#[derive(Default)]
-struct CommitToFilter {
-    // map[peer_id] -> committed index.
-    committed: Arc<Mutex<HashMap<u64, u64>>>,
-}
+#[test]
+fn test_replica_read_on_stale_peer() {
```

Comment:
I don't think people can guess what it's actually testing, better add some comment.

Reply:
Done